src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -163,12 +163,14 @@ class TensorRTRuntime : public JSONRuntimeBase {
           const std::string name = nodes_[nid].GetOpName() + "_" + std::to_string(j);
           int binding_index = engine->getBindingIndex(name.c_str());
           ICHECK_NE(binding_index, -1);
+#if TRT_VERSION_GE(6, 0, 1)
```

Comment:
Do we need a warning or something when TRT < 6.0.1 and `use_implicit_batch_ == False`?

Reply:
Below this code on [line 215](https://github.com/apache/tvm/blob/ceda892f081ceb39457394916a18f6e7045a1a8e/src/runtime/contrib/tensorrt/tensorrt_runtime.cc#L215) we just ignore use_implicit_batch_ flag if TRT <6.0.1.
This PR applies the same logic.

Reply:
I see. Make sense.